lib/service/film_manager.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:async';
+import 'package:my_movies/model/film.dart';
+import 'package:my_movies/widget/rating_button.dart';
+import 'package:rxdart/rxdart.dart';
+
+class FilmManager {
+  final _filmStreamController = StreamController<List<Film>>.broadcast();
+
+  List<Film> films = [];
```

Comment:
Tato proměnná by zde neměla být. Nahradil bych _filmStreamController pomocí BehaviorSubject z RxDart. Ten má v sobě getter na získání aktuální hodnoty z něj.

Odůvodnění: Duplikace stavu, films i _filmStreamController v sobě obsahují stejná data, takže musíš 100% zarušit, že jsou synchronizovány.

Note: Proměnná by měla být final